core/src/test/scala/unit/kafka/server/ClientQuotasRequestTest.scala

```diff
@@ -17,6 +17,8 @@
 
 package kafka.server
 
+import java.util
```

Comment:
This should go with the other java imports, which I guess are combined in this file